src/scripts/main.js

```diff
@@ -1,8 +1,29 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const person of people) {
```

Comment:
Consider using a functional programming method like 'forEach' instead of a traditional loop for iterating over the 'people' array.